.eslintrc

```diff
@@ -74,6 +75,20 @@
     "object-curly-spacing": [2, "always"],
     "quotes": [2, "single", "avoid-escape"],
     "radix": 2,
+    "react/display-name": 0,
+    "react/jsx-quotes": [2, "double", "avoid-escape"],
```

Comment:
Can we change this to `[2, "single", "avoid-escape"]` to be inline with our standard js rules